demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>There seems to be an image here that neither ReviewNB nor Github's rendering can show https://github.com/stellargraph/stellargraph/blob/6c7d2e09681ccdbd3afc1e56a872680cc6fc5523/demos/graph-classification/supervised-graph-classification.ipynb, although nbviewer.jupyter.org can: https://nbviewer.jupyter.org/github/stellargraph/stellargraph/blob/6c7d2e09681ccdbd3afc1e56a872680cc6fc5523/demos/graph-classification/supervised-graph-classification.ipynb</p><p> </p><p>I wonder if we can find a different way to do this that Github can handle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I don't know how github displays images or why it fails on this one specifically.

We can just remove it since the architecture is clearly explained in the text and easily understood from reading `create_graph_classification_model`. 

That said, I'll have a look if the problem is with the image format.

Reply:
I added the image to the repo and it looks like it is rendering correctly now.

Reply:
Hm, unfortunately I don't see it: https://github.com/stellargraph/stellargraph/blob/ca295c3e9a8dcc61fe7570cb765a7a7f543a85c0/demos/graph-classification/supervised-graph-classification.ipynb

![image](https://user-images.githubusercontent.com/1203825/76820893-86722c80-6860-11ea-9532-f49597617328.png)

One solution would be to include the image as you originally had it, but tweak the text to not rely on it so that it's fine if it doesn't render (I think this may be as simple as removing the "The below image shows the architecture we are going to use." sentence, since, as you say, the architecture is explained clearly in the text).